eclair-core/src/test/scala/fr/acinq/eclair/payment/MultiPartPaymentFSMSpec.scala

```diff
@@ -110,6 +111,19 @@ class MultiPartPaymentFSMSpec extends TestKitBaseClass with AnyFunSuiteLike {
     f.eventListener.expectNoMsg(50 millis)
   }
 
+  test("fail all if total pay-to-open is below minimum") {
+    val f = createFixture(250 millis, 20000000 msat)
```

Comment:
It would be useful to always have an `assert` that the `minPayToOpenAmount` is higher than `4000000 msat` but lower than `20000000 msat` since it's not explicitly set by this test (to make sure that if we change the internals of `createPayToOpenPart` we can't regress here without detecting it).

Reply:
Done in c50d7f7.